### PR TITLE
docs: update folder capitalisation preferred practices

### PR DESCRIPTION
**Pull request by patrinoua**
The type of this PR is: **docs**

This PR implements the results of [this](https://artsy.slack.com/archives/C02BAQ5K7/p1643294097102400) vote.

This PR resolves []

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

### Changelog updates

ⓘ 'User-facing' changes will be published as release notes.

#### Cross-platform user-facing changes

- Rename modal title - patrinoua

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 556a183c48c0c81807d866fa83bf74e8b6597f61
</p>

File: docs/preferred_practices.md [hunk 1]
```diff
@@ -72,9 +72,46 @@ touch .i-am-helping-out-with-the-strictness-migration
 
 ### Keep File Structure Organized (in progress)
 
-Everything in `src/` is React Native. Within this folder, things can be a bit of a mess. We are working on cleaning it up. Check back here for more later.
+Everything in `src/` is React Native. Within this folder things can be a bit of a mess and we are working on improving that.
 
-**TODO**: Figure out what we want our directory structure to be and define it here.
+According to the React convention we capitalise **Components and Component Folders**, but keep everything else lowercase.
+
+For example migrations, routes, state would be folders with small letter while MyComponent.tsx would start with a Capital letter.
```
Comment (pvinis) [inline docs/preferred_practices.md:79]:
```suggestion
For example `mutations`, `routes`, `state` would be **lowercase** folders, while `MyComponent.tsx` would be a **PascalCase** file.
```

File: docs/preferred_practices.md [hunk 2]
```diff
@@ -72,9 +72,46 @@ touch .i-am-helping-out-with-the-strictness-migration
...
+│   ├── state
+│   |  ├── stateFunction.ts
+│   ├── utils
+│   |  ├── utilFunction.ts
+├── …
+```
+
+Another example is:
+
+If we have a buttons folder which has many ButtonFile.tsx inside we keep a small letter.
```
Comment (pvinis) [inline docs/preferred_practices.md:96]:
```suggestion
If we have a `buttons` folder which has many ButtonFile.tsx inside we keep it **lowercase**. 
```

File: docs/preferred_practices.md [hunk 3]
```diff
@@ -72,9 +72,46 @@ touch .i-am-helping-out-with-the-strictness-migration
...
+If we have a buttons folder which has many ButtonFile.tsx inside we keep a small letter. 
+
+```
+├── buttons
+│   ├── RedButton.tsx
+│   ├── GreenButton.tsx
+│   ├── YellowButton.tsx
+├── …
+```
+However, if we have a Button folder which only exports Button.tsx we write that with a capital letter.
```
Comment (pvinis) [inline docs/preferred_practices.md:105]:
```suggestion
However, if we have a `Button` folder which exports only one button component, we write that with in **PascalCase**.
```

File: docs/preferred_practices.md [hunk 4]
```diff
@@ -72,9 +72,46 @@ touch .i-am-helping-out-with-the-strictness-migration
...
+
+```
+├── Button
+│   ├── Button.tsx
+│   ├── Button.tests.tsx
```
Comment (pvinis) [inline docs/preferred_practices.md:110]:
```suggestion
│   ├── Button.tests.tsx
│   ├── hooks
│   ├── animations
```

File: docs/preferred_practices.md [hunk 5]
```diff
@@ -72,9 +72,46 @@ touch .i-am-helping-out-with-the-strictness-migration
...
+
+```
+
+TODO: Add a migration guide?
```
Comment (pvinis) [inline docs/preferred_practices.md:114]: i dont think we need a migration guide. like you said, lets let past be past :D

**Review by pvinis — CHANGES_REQUESTED** [review]
made some changes but leaving it for you to commit these in.

Once we have the rules set, lets pair and take a look at the files/folders we have now to make sure we didn't miss anything, and then we can merge.

PS. I made this into a draft, while we are working on it, if you dont mind.

File: docs/preferred_practices.md (hunk 5 above)
Comment (patrinoua) [reply docs/preferred_practices.md:114]: For sure. How about the things we touch in the future though? 
Wouldn't it be nice to have a way to bring them to our modern ways?
Perhaps we could use something 

`State -> stateMigration -> state`

**Review by pvinis — APPROVED** [review]

**Pull request merged**